src/libbson/tests/json/bson_corpus/top.json

```diff
@@ -69,11 +84,11 @@
     "parseErrors": [
         {
             "description" : "Bad $regularExpression (extra field)",
-            "string" : "{\"a\" : \"$regularExpression\": {\"pattern\": \"abc\", \"options\": \"\", \"unrelated\": true}}}"
+            "string" : "{\"a\" : {\"$regularExpression\": {\"pattern\": \"abc\", \"options\": \"\", \"unrelated\": true}}}"
```

Comment:
Related to https://github.com/mongodb/specifications/commit/baee952aca3d0d2515866f17e3f2c79d73840d09. I don't believe there was a corresponding DRIVERS ticket or any downstream language tickets for this.